### PR TITLE
Add header-only library cereal

### PR DESCRIPTION
**Pull request by saraedum**

### CODE REVIEW

**Comment by saraedum** [comment]
@conda-forge/help-c-cpp this PR is ready for review :)

File: recipes/cereal/bld.bat [hunk 1]
```diff
@@ -0,0 +1 @@
+xcopy /s /f include %LIBRARY_INC%
```
Comment (xhochy) [inline recipes/cereal/bld.bat]: `noarch: generic` package builds are just run on Linux, no need for this file.

File: recipes/cereal/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+source:
+  url: https://github.com/USCiLab/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: 1921f26d2e1daf9132da3c432e2fd02093ecaedf846e65d7679ddf868c7289c4
+
+build:
+  noarch: generic
+  number: 0
+
+requirements:
+  build:
```
Comment (xhochy) [inline recipes/cereal/meta.yaml]: You can also add an empty list normally.
```suggestion
  build: []
```

Comment (xhochy) [reply recipes/cereal/meta.yaml]: Ah, I see on gitter that this doesn't work ;)

File: recipes/cereal/bld.bat (hunk 1 above)
Comment (saraedum) [reply recipes/cereal/bld.bat]: So, the CI will fail here and that's not a problem then.

**Comment by saraedum** [comment]
@xhochy Thanks for having a look. I removed the extra `bld.bat`. So this is ready for review again.

**Comment by xhochy** [comment]
It should not fail on the feedstock. Failure for Win on staged-recipes is fine.

**Pull request merged**